src/main/java/org/codehaus/groovy/classgen/ClassCompletionVerifier.java

```diff
@@ -418,10 +418,6 @@ private void checkMethodsForIncorrectModifiers(final ClassNode cn) {
                 addError("The " + getDescription(method) + " from " + getDescription(cn) +
```

Comment:
In an interface, is final allowed on private or static methods?  If allowed, this check needs a second look.  The message may need to check for method with and without body before stating "abstract by definition".

Reply:
I don't think there is any case where final is allowed.